src/options/options.js

```diff
@@ -7,6 +7,8 @@ const _redirectUrlField = document.getElementById("redirecturl")
 const _appidField = document.getElementById("appid")
 const _endpointField = document.getElementById("endpoint")
 
+const saveableFields = [ _endpointField, _appidField ]
```

Comment:
hey what?! put this back I liked it the other way

Reply:
:trollface: 
so uh `document.getElementByTagName("input").on("blur", saveOptions)` ?